CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.5.0] - 2024-07-08
+
+### 📚 Documentation
+
+- Add doc comments for tproxy
```

Comment:
@jwhb Hmm, could it be that `feat:` commits are not getting picked up git-cliff at the moment? 🤔

Reply:
What is strange about this is that running `git-cliff` locally actually generates the changelog as you would expect... Not sure why the GitHub Action is generated an incorrect version, as it is clearly picking up that there has been at least one `feat:` commit that implies bumping the crate's minor version.

Reply:
Hmm, seems to me as if the dependency updates are also not picked up by the GitHub Action anymore 🤔